# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/plan9port/cross-disable-web-manual.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- plan9port-d0d4408/INSTALL.vanilla	2022-01-09 18:18:09.495000000 +0100
+++ plan9port-d0d4408/INSTALL	2022-01-09 18:18:57.263000000 +0100
@@ -184,7 +184,7 @@
 		cd $PLAN9
 		sh lib/moveplan9.sh
 		echo "* Building web manual..."
-		(
+		[ "$PLAN9" -eq "$HOSTPLAN9" ] && (
 			cd $PLAN9/dist
 			echo cd `pwd`';' mk man
 			mk man
